### PR TITLE
Merge and Email activities

### PR DESCRIPTION
**Pull request by sethyates**

### CODE REVIEW

**Comment by sethyates** [comment]
Fixes #89 and #90.

File: scripts/test-email-file.sh [hunk 1]
```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
```
Comment (realstraw) [inline scripts/test-email-file.sh]: how do we run these tests?

Comment (sethyates) [inline scripts/test-email-file.sh]: ./scripts/test-email-file.sh ;-) they can be run on your dev box as it mocks out the EC2 requirements.

Comment (realstraw) [inline scripts/test-email-file.sh]: Cool

**Comment by realstraw** [comment]
:+1:

**Pull request closed without merging**